tests/ui/closures/2229_closure_analysis/match/issue-88331.stderr

```diff
@@ -10,13 +10,14 @@ note: `Opcode` defined here
 LL | pub struct Opcode(pub u8);
    |            ^^^^^^
    = note: the matched value is of type `Opcode`
-help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern, a match arm with multiple or-patterns as shown, or multiple match arms
+help: ensure that all possible cases are being handled by adding more match arms
    |
 LL ~         Opcode::OP1 => unimplemented!(),
-LL ~         Opcode(0_u8) | Opcode(2_u8..=u8::MAX) => todo!(),
+LL +         Opcode(0_u8) => { todo!() },
```

Comment:
Note that while this is also my personal preference, to put a `,` after blocks in match arms, rustfmt's defaults are to omit it.

Reply:
I agree this should be reverted. If you want to make this change separately, break it out into another PR.

Reply:
I've reverted the commas.